### PR TITLE
Introduce wheel build workflow for release

### PR DESCRIPTION
**Pull request by bearzx**
In this PR I added another GHA for the beta release. It's mostly copied from `nightly-wheel.yml` so some refactoring or unification work can be forseen in the future. But here I wanted it to start simple and separate.

The workflow will checkout the release branch, build, run the criteo integration test, then upload it to github so it can be retrieved to upload to pypi.

TODO: since `release/0.1.0` is already created, after this PR is merged to `main`, I'll need to merge the `main` branch to `release/0.1.0`.

### CODE REVIEW

**Comment by wenleix** [comment]
Consider to use `git commit --amend` and do force push unless it's some "major changes" that you want to keep the commit history :)

**Comment by bearzx** [comment]
> Consider to use `git commit --amend` and do force push unless it's some "major changes" that you want to keep the commit history :)

cool yeah I used the `--amend` arg once but it still appeared in the history, now I know I forgot the force push part :D

File: packaging/manylinux/python_helper.sh [hunk 1]
```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+python_nodot="$(echo $PYTHON_VERSION | tr -d '.')"
```
Comment (wenleix) [inline packaging/manylinux/python_helper.sh:8]: @ejguan : Seems TorchData has similar stuff. What are they used to? -- didn't remember seeing simliar thing in domain libraries.

**Review by wenleix — APPROVED** [review]
LGTM. Thanks1

File: packaging/manylinux/python_helper.sh (hunk 1 above)
Comment (bearzx) [reply packaging/manylinux/python_helper.sh:8]: Looks like it updates PATH to reflect the right python? I'll test more to see if this is necessary in our case after this PR lands.

**Pull request closed without merging**